pkgs/by-name/nx/nxwitness-client/package.nix

```
@@ -0,0 +1,119 @@
+{
+  stdenv,
+  lib,
+  autoPatchelfHook,
+  buildFHSEnv,
+  dpkg,
+  fetchurl,
+  glib,
+  gst_all_1,
+  libGL,
+  libgudev,
+  libudev-zero,
+  libxcb,
+  libxkbfile,
+  libxml2,
+  libxslt,
+  openal,
+  qt6Packages,
+  wayland,
+  xcbutilimage,
+  xcbutilkeysyms,
+  xcbutilrenderutil,
+  xcbutilwm,
+}:
+let
+  version = "6.0.3";
+  build = "40736";
+
+  libxml2_13 = libxml2.overrideAttrs rec {
+    version = "2.13.8";
+    src = fetchurl {
+      url = "mirror://gnome/sources/libxml2/${lib.versions.majorMinor version}/libxml2-${version}.tar.xz";
+      hash = "sha256-J3KUyzMRmrcbK8gfL0Rem8lDW4k60VuyzSsOhZoO6Eo=";
+    };
+  };
+
+  buildInputs = [
+    glib
+    gst_all_1.gst-plugins-base
+    gst_all_1.gstreamer
+    libGL
+    libgudev
+    libudev-zero
+    libxcb
+    libxkbfile
+    libxml2_13
+    libxslt
+    openal
+    qt6Packages.qtbase
+    qt6Packages.qtquicktimeline
+    qt6Packages.qtwayland
+    qt6Packages.qtwebengine
+    qt6Packages.qtwebsockets
+    qt6Packages.qtwebview
+    wayland
+    xcbutilimage
+    xcbutilkeysyms
+    xcbutilrenderutil
+    xcbutilwm
+  ];
+
+  meta = {
+    description = "Desktop Client for Nx Witness Video Systems";
+    homepage = "https://nxvms.com/";
+    sourceProvenance = with lib.sourceTypes; [ binaryNativeCode ];
+    license = lib.licenses.unfree;
+    maintainers = with lib.maintainers; [ netali ];
+    platforms = [ "x86_64-linux" ];
+  };
+
+  nxwitness_client = stdenv.mkDerivation (finalAttrs: {
+    inherit buildInputs meta;
+    pname = "nxwitness-client";
+    version = "${version}.${build}";
+
+    src = fetchurl {
+      url = "https://updates.networkoptix.com/default/${build}/linux/${finalAttrs.pname}-${finalAttrs.version}-linux_x64.deb";
```

```suggestion
      url = "https://updates.networkoptix.com/default/${build}/linux/nxwitness-client-${finalAttrs.version}-linux_x64.deb";
```

https://github.com/NixOS/nixpkgs/issues/310373

---

Fixed.